### DISCUSSION TITLE
RTranslator 2.1.0

### DISCUSSION THREAD

**Discussion by niedev** [niedev/RTranslator#69] [General]
What's new:

- New GUI! Designed by [Chiara Chindamo](https://www.linkedin.com/in/chiara-chindamo-946053234/).
- Added speak and copy buttons to the text translation mode.
- Added the option to manually control the mics in WalkieTalkie mode.
- Added the option to use low-quality languages.
- Separated the languages of the text translation from the ones of WalkieTalkie and Conversation.
- Added Ukrainian localization (thanks to @OriginalSimon)
- Corrected Chinese localization (thanks to @Agiens02)
- Integrated the [GalleryImageSelector](https://github.com/niedev/GalleryImageSelector) code in the app (thanks to @Agiens02)
- Fixed bugs

<hr /><em>This discussion was created from the release <a href='https://github.com/niedev/RTranslator/releases/tag/2.1.0'>RTranslator 2.1.0</a>.</em>

**Comment by PawelGu** [comment]
I finally got a new mobile with 12 GB RAM and can use the app to it's fullest. It's working great so far and the improvements are very noticeable - Thank you ~~~~~~
